.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -1,49 +1,26 @@
 ---
 name: Bug report
-about: Create a report to fix a bug
+about: Create an issue to fix a bug
 ---
 
 <!--
 BEFORE SUBMITTING: Please search to make sure this issue has not been opened already
 -->
 
-### Title/Description
-<!--
-A clear and concise description of what the bug is.
--->
+## Description
 
-### Environment
- - Browser:
+## Environment
+ - Browser: Chrome
+ - Wallet: MetaMask
```

Comment:
Why setting this default values?

Reply:
To not type them. Or what do you mean?

Reply:
I use Firefox as main browser, so I would have to delete them every time :smile: 
But it's fine

Reply:
Let me remove it then 😅